app/components/access_panels/location_item_component.html.erb

```diff
@@ -10,7 +10,7 @@
     </td>
   <% end %>
   <% if render_item_details? %>
-    <td class="item-availability" data-live-lookup-id="<%= document.live_lookup_id %>" data-status-target=".availability-icon" data-item-id="<%= item.live_lookup_item_id if item.live_status? %>" <%= "data-request-url='#{helpers.request_url(document, library: item.library, location: item.home_location, barcode: item.barcode)}'".html_safe if render_real_time_availability_request_link? %>>
+    <td class="item-availability" data-live-lookup-id="<%= item.live_lookup_instance_id %>" data-status-target=".availability-icon" data-item-id="<%= item.live_lookup_item_id if item.live_status? %>" <%= "data-request-url='#{helpers.request_url(document, library: item.library, location: item.home_location, barcode: item.barcode)}'".html_safe if render_real_time_availability_request_link? %>>
```

Comment:
Do we also need to change this line? https://github.com/sul-dlss/SearchWorks/blob/master/app/components/access_panels/at_the_library_component.html.erb#L2 ?

Reply:
I don't think so (although maybe it's redundant? Or we should be omitting item-level attributes if it's the same as that?)